packages/utils/src/facility/bulk-import-facility.ts

```diff
@@ -141,15 +138,67 @@ async function main({ cxId, inputPath, dryrun }: FacilityImportParams) {
     readFileFromLocal(inputPath, parser).catch(reject);
   });
 
-  await Promise.all(rowPromises);
+  return rows;
+}
 
-  await fs.writeFile(payloadCreatesFilePath, JSON.stringify(createdFacilities, null, 2), "utf8");
+async function processRow(
+  row: InputRowFacilityImport,
+  cxId: string,
+  isDryRun: boolean,
+  logsFilePath: string
+): Promise<FacilityInternalDetails | null> {
```

Comment:
no `null` on our codebase if we can avoid, please - prefer `undefined` as stated on the code standards

here and in other places on this PR